CMakeLists.txt

```diff
@@ -12,5 +12,6 @@ enable_testing()
 add_subdirectory(gtest)
 
 add_executable(zmij-test zmij-test.cc)
+target_compile_features(zmij-test PRIVATE cxx_std_20)
```

Comment:
This looks unused.

Reply:
We only need C++17. I removed the only C++20 feature (bit_cast) earlier.

Reply:
well sure, then it's C++17, I missed that change.